src/numberGenerator.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const { SECRET_SIZE } = require('./constants');
+
+const generateNumber = () => Math.floor(Math.random() * 10);
+
+const generateSecretNumbers = () => (
+  Array.from({ length: SECRET_SIZE }, _ => generateNumber())
+);
```

Comment:
Check requirements, number should have 4 **different** digits, make sure, your numbers doesn't repeat.